Aardvark.podspec

```diff
@@ -1,12 +1,12 @@
 Pod::Spec.new do |s|
   s.name     = 'Aardvark'
-  s.version  = '1.5.0'
+  s.version  = '2.0.0'
   s.license  = 'Apache License, Version 2.0'
   s.summary  = 'Aardvark is a library that makes it dead simple to create actionable bug reports.'
   s.homepage = 'https://github.com/square/Aardvark'
   s.authors  = 'Square'
   s.source   = { :git => 'https://github.com/square/Aardvark.git', :tag => s.version }
-  s.source_files = 'Aardvark/*.{h,m}', 'Categories/*.{h,m}', 'Other/*.{h,m}', 'Logging/*.{h,m}', 'Log Viewing/*.{h,m}', 'Bug Reporting/*.{h,m}'
+  s.source_files = 'Aardvark/*.{h,m,swift}', 'Categories/*.{h,m,swift}', 'Other/*.{h,m,swift}', 'Logging/*.{h,m,swift}', 'Log Viewing/*.{h,m,swift}', 'Bug Reporting/*.{h,m,swift}'
```

Comment:
For ease of adoption, and since we're introducing breaking changes in 2.0 I think it's fine to keep this as-is.

Good thought though, and TIL that was possible.